### ISSUE TITLE
Include more keys in JamfConnect Login profile

### ISSUE THREAD

**Issue by robcraigfl** [ProfileCreator/ProfileManifests#431]
Would be great to have more keys included in the JamfConnect Profile. Like <key>LocalFallback</key> and others. Also to be able to edit this by hand to add anything that is missing.

**Comment by robcraigfl** [comment]
Thanks for the reply. For some reason when I tried editing it in a text editor It did not work. I've just retried and it's working now. 

Here is the definitive list for the keys: https://docs.jamf.com/jamf-connect/1.18.2/administrator-guide/Configuring_Jamf_Connect_Login_with_Microsoft_Azure_AD.html

**Comment by robcraigfl** [comment]
I've just tested editing in BBEDit the config file and then importing back to ProfileCreator and it breaks the profile. A working profile now fails when deployed :( Is there any other way to add the LocalFallBack key in?

**Comment by kevinmcox** [comment]
Why are you importing it back into PC after the manual edit?

**Comment by robcraigfl** [comment]
Because I need to sign it :( And for some reason I can only sign via ProfileCreator. Jamf does not accept anything when I sign it via CLI :(

**Comment by kevinmcox** [comment]
Ah, there is a macOS bug with signing on the command line. Try using Hancock: https://github.com/JeremyAgost/Hancock

**Comment by robcraigfl** [comment]
Amazing. Works with Hancock.

**Comment by kevinmcox** [comment]
`com.jamf.connect.login-OpenID.plist`

Latest docs: https://docs.jamf.com/jamf-connect/2.9.0/documentation/Login_Window_Preferences.html

**Comment by apizz** [comment]
There's some reworking of the existing manifests which I already started working on. Don't use these so will need help to vet and test

**Comment by kevinmcox** [comment]
Glad some progress has been made. I started looking at it and there are a lot of keys... (We don't use Jamf Connect either.)

**Comment by apizz** [comment]
Yeah not only that but a lot has changed since original contribution and there's lots of little things to fix so it's slow going...
